### DISCUSSION TITLE
Move grate extension to Cake Contrib

### DISCUSSION THREAD

**Discussion by franhoey** [cake-build/cake#4152] [Extension requests]
This is still WIP but has enough I think to be of value if I was to drop dead!

The extension will be to enable to [grate ](https://erikbra.github.io/grate/) to be used in a Cake script. Grate is a replacement for RoundhousE designed to be backwards compatabile (mostly) with RoundhousE. This extension will do the same.

https://github.com/franhoey/Cake.grate

**Comment by augustoproiete** [comment]
Hey @franhoey here are the instructions for the transfer of your repo:

For GitHub, this is a multi-step operation due to how GitHub manages permissions for organizations.

1. Transfer the repo to my user augustoproiete ([instructions](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository))
2. I transfer it to the [Cake Contrib](https://github.com/cake-contrib) organization
3. You accept the invite to join the organization (you should have received an email)
4. I add you as admin in the repo, within the org

For NuGet, open the "Manage" package UI for your packages on [nuget.org](http://nuget.org/) and add the user [cake-contrib](https://www.nuget.org/profiles/cake-contrib) as owner

**Reply by franhoey** [reply]
Thanks @augustoproiete I've transferred the Repo to you now

**Reply by augustoproiete** [reply]
Great. Thanks @franhoey 
The repo has been moved to https://github.com/cake-contrib/Cake.grate and you should have Admin access to it.

Let us know if you run into any issues.

**Reply by franhoey** [reply]
@augustoproiete The first version of the extension is ready, I'm just sorting the build automation and noticed you've created a Cake.grate nuget package for this already. 
Am I right to be building a github action (using Cake) for the deploy, or is there another process?

**Reply by augustoproiete** [reply]
Hey @franhoey yes, GitHub actions is the way to go, and what we've been using for Cake add-ins in recent years.

Send me your NuGet username, so I can transfer the Cake.grate NuGet Package to you

**Reply by franhoey** [reply]
Continued on this [thread](https://github.com/orgs/cake-build/discussions/4287)
